Assets/Scripts/Checkpoint.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Checkpoint : MonoBehaviour
+{
+    [SerializeField]
+    private Color inactiveColor, activeColor;
+    private AudioSource audioSource;
```

Comment:
It's generally a good idea to leave a blank line between your SerializeFields and non-SerializeFields, so you might consider adding whitespace here